core/src/scss/components/hero/_hero--caption.scss

```diff
@@ -31,15 +31,24 @@
     @include modular-spacing(padding-bottom, 2);
     text-align: right;
     display: block;
+    clear: both;
 
     p {
       @include credits;
     }
+  }
 
-    // Only allow for 60% width on large screen sizes.
-    @include grid-media('lg') {
-      @include padding(null null null 40%);
-    }
+  .su-hero__content-inner {
+    float: right;
 
+    @include grid-media("lg") {
+      width: ceil($su-screen-lg * 0.6);
+    }
+    @include grid-media("xl") {
+      width: ceil($su-screen-xl * 0.6);
+    }
+    @include grid-media("2xl") {
+      width: ceil($su-screen-2xl * 0.6);
```

Comment:
Are we using fixed width container and not fluid 60%? I'm totally fine with this - looks close enough. Just want to double check because it's a little different from the request.